streamingpro-mlsql/src/main/java/streaming/dsl/TrainAdaptor.scala

```diff
@@ -76,6 +77,10 @@ class TrainAdaptor(scriptSQLExecListener: ScriptSQLExecListener) extends DslAdap
       options = options ++ Map("__dfname__" -> tableName)
     }
 
+    if(!skipAuth() && sqlAlg.isInstanceOf[ETAuth]){
```

Comment:
It seems the mode will be 'train' forever. However, it can be run/predict/train according to the different statements.